### ISSUE TITLE
remove_nested_fields_link should not appear in first form

### ISSUE THREAD

**Issue by thiyagarajan** [ncri/nested_form_fields#43]
Hi, 

We needed remove_nested_fields_link should not appear in first form.

How we can remove remove_nested_fields_link should not appear in first fom?

Thanks

**Comment by ncri** [comment]
Have you tried hiding it with css or javascript?

**Comment by JoeWoodward** [comment]
+1 for a ruby based configuration

**Comment by ncri** [comment]
Yeah, would be nice, I agree. I might look into it when I have some time. Pull requests welcome. ;-)
